Description: Don't embed the build date
 Debian strives for reproducible builds. Stop using the __DATE__ and __TIME__
 macros which shown when running "elinks --version".
Author: Moritz Muehlenhoff <jmm@debian.org>

Index: elinks/src/main/version.c
===================================================================
--- elinks.orig/src/main/version.c	2017-12-01 11:07:37.573697705 +0100
+++ elinks/src/main/version.c	2017-12-01 11:07:37.553697625 +0100
@@ -103,8 +103,6 @@
 	}
 
 	add_char_to_string(&string, '\n');
-	add_format_to_string(&string, _("Built on %s %s", term),
-			     build_date, build_time);
 
 	if (more) {
 		add_to_string(&string, "\n\n");
Index: elinks/src/vernum.c
===================================================================
--- elinks.orig/src/vernum.c	2017-12-01 11:07:37.573697705 +0100
+++ elinks/src/vernum.c	2017-12-01 11:07:37.569697689 +0100
@@ -8,6 +8,4 @@
 
 #include "vernum.h"
 
-unsigned char *build_date = __DATE__;
-unsigned char *build_time = __TIME__;
 unsigned char *build_id = BUILD_ID;
